ui/admin/tests/acceptance/aliases/selectors.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+export const FIELD_DESTINATION_ID_SELECTOR = '[name="destination_id"]';
+export const FIELD_HOST_ID_SELECTOR = '[name="authorize_session_arguments"]';
```

Comment:
`_SELECTOR` can be removed from these selectors since it is implied when importing from a selectors file.

Reply:
Is the general rule that we append `_SELECTOR` if the const is not in a selectors file, otherwise we do not append?

Reply:
Yes that is correct.